src/bin/rav1e.rs

```diff
@@ -265,6 +265,31 @@ fn init_logger() {
   pretty_env_logger::init();
 }
 
+cfg_if::cfg_if! {
+  if #[cfg(target_os = "windows")] {
+    fn print_rusage() {
+      eprintln!("Windows is not supported.");
+    }
+  } else {
+    fn print_rusage() {
+      let (utime, stime, maxrss) = unsafe {
+        let mut usage = std::mem::zeroed();
+        let _ = libc::getrusage(libc::RUSAGE_SELF, &mut usage);
+        (usage.ru_utime, usage.ru_stime, usage.ru_maxrss)
+      };
+      eprintln!(
+        "utime: {} us",
+        utime.tv_sec as i64 * 1_000_000 + utime.tv_usec as i64
```

Comment:
If this is benchmarking a whole encode, it probably makes more sense to show something like "2345.006 sec" than "23450067890 us".

Reply:
Formatted.